servatrice/servatrice.ini.example

```diff
@@ -107,6 +107,9 @@ allowedpunctuation=_.-
 ; If a username can begin with punctuation defined in allowedpunctuation
 allowpunctuationprefix=false
 
+; Disallow usernames containing these words
+disallowedwords="admin,administrator"
```

Comment:
Should there be a default other than admin that I should add while I'm fixing that up?